sdcm/nemesis.py

```diff
@@ -227,6 +227,21 @@ def disrupt_nodetool_decommission(self, add_node=True):
                     self.cluster.wait_for_init(node_list=new_nodes)
                     self.reconfigure_monitoring()
 
+    def disrupt_nodetool_enospc(self, sleep_time=30):
+        self._set_current_disruption('Enospc %s' % self.target_node)
+        if not os.path.ismount('/var/lib/scylla'):
+            self.log.error("Scylla doesn't use an individual storage, skip enospc test")
+            return
+        dd_cmd = 'dd if=/dev/zero of=/var/lib/scylla/occupy_rest_space'
+        self.log.debug('Cost rest space on /var/lib/scylla/ by {}'.format(dd_cmd))
+        process.run(dd_cmd, ignore_status=True)
+        assert result.exit_status == 1 and 'No space left on device' in result.stderr
+        self._run_nodetool('nodetool status', self.target_node)
+        time.sleep(sleep_time)
```

Comment:
I think we need to wait for db_down.
When Scylla is out of space it drains the host and take it down.
So, we need to make sure it's down and not serving requests, then to free up the space and restart scylla.